src/akamai/js/akamaiHD_flash.js

```diff
@@ -1194,7 +1194,7 @@ var swfobject = function() {
             }
           }
         }
-        for (var n in parObj) {
+        for (var n = 0; n < parObj.length; n++) {
           if (parObj[n] != Object.prototype[n] && n.toLowerCase() != "movie") { // filter out prototype additions from other potential libraries and IE specific param element
             createObjParam(o, n, parObj[n]);
```

Comment:
Has been changed some days back, githib is failing to update the conversation page to reflect the current code...